log2ram.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Log2Ram
 DefaultDependencies=no
-Before=basic.target rsyslog.service syslog.target systemd-journald.service sysinit.target shutdown.target apache2.service
+Before=basic.target rsyslog.service syslog.target systemd-journald.service sysinit.target shutdown.target zram-swap-conf.service apache2.service
```

Comment:
Is this service a standard ?

Reply:
Dunno to be honest I was struggling as the Log2Ram of IgnoreOnIsolate=yes seems to cause me loads of probs.
Doesn't have to be there but I am going to leave it up to you.
I was originally splitting zram creation around the wait routine but each instruction is now continuous and there is little problem with either apart from the overrides of  IgnoreOnIsolate=yes and was going with the flow rather than understanding why log2ram has so many before/after/conflicts and ignores?
You can answer that one and change as needed.